tests/lint/test.sh

```diff
@@ -33,18 +33,33 @@ rlJournalStart
         rlAssertGrep "plans/bad" $rlRun_LOG
         rlAssertGrep "stories/bad" $rlRun_LOG
         # linting story
-        rlAssertGrep "warn summary should not exceed 50 characters" $rlRun_LOG
-        rlAssertGrep "fail unknown attribute 'exampleee' is used" $rlRun_LOG
+        rlAssertGrep "warn C000 fmf node failed schema validation" $rlRun_LOG
+        rlAssertGrep "pass C001 summary key is set" $rlRun_LOG
+        rlAssertGrep "warn C002 summary should not exceed 50 characters" $rlRun_LOG
+        rlAssertGrep "fail S001 unknown key \"exampleee\" is used" $rlRun_LOG
+        rlAssertGrep "pass S002 story key is defined" $rlRun_LOG
         # linting test
-        rlAssertGrep "pass test script must be defined" $rlRun_LOG
-        rlAssertGrep "pass directory path must be absolute" $rlRun_LOG
-        rlAssertGrep "pass directory path must exist" $rlRun_LOG
-        rlAssertGrep "warn summary is very useful for quick inspection"   \
-            $rlRun_LOG
-        rlAssertGrep "fail unknown attribute 'summarrry' is used" $rlRun_LOG
+        rlAssertGrep "warn C000 fmf node failed schema validation" $rlRun_LOG
+        rlAssertGrep "warn C001 summary key is missing, it is useful for quick inspection" $rlRun_LOG
+        rlAssertGrep "skip C002 summary key is missing, it is useful for quick inspection" $rlRun_LOG
```

Comment:
this warn/skip combo looks confusing to me, why it is designed this way? also the checks look the same, at least the text is the same.

Reply:
We can improve those. There are two distinct checks:

```shell
$ tmt test lint --list-checks
Linters available for tests
...
C001: summary key should be set, it is very useful for quick inspection
C002: summary should not exceed 50 characters
```

When the summary key is not set at all - reported as a warning, `warn C001 summary key is missing`, the second test is skipped. I can change it to a warning, but it sort of makes sense to me, as there's no `summary` to check for length.

I'm open to suggestions :)

Reply:
So on brief look at lint report I was thinking: Why the same check is reported twice under different numbers with different results?

Reply:
As discussed, C001 and C002 are now a single check, fixed in 8b4d9ce2:

```
/stories/features/report-result
warn C001 summary key is missing

/stories/features/prepare/all
pass C001 summary key is set and is reasonably long

/stories/features/prepare/minimal
warn C001 summary should not exceed 50 characters
```

Reply:
thank you.